web/template/watch.gohtml

```diff
@@ -200,18 +200,18 @@
             <div class="flex justify-between align-middle mt-2 md:hidden">
                 {{template "actions" . }}
             </div>
-            {{if $stream.Description}}
-                {{$isMinLen := gt (len $stream.Description) 128}}
-                <div x-data="{less: {{$isMinLen}}}" class="text-3 p-4 border-t-2 my-3 border-secondary video-description">
-                    <span class="font-semibold">Video Description:</span>
-                    <div :class="less && 'truncate-children'">{{.Description}}</div>
-                    <template x-if="{{$isMinLen}}">
-                        <button class='text-xs text-gray-500 dark:text-gray-400 uppercase mt-2'
-                                @click="less = !less" x-text="less ? 'show more' : 'show less'">show less
-                        </button>
-                    </template>
+            <div x-data="{descriptionlen: `{{len .Description}}`, less:true}"
+                 class="text-3 p-4 border-t-2 my-3 border-secondary video-description">
```

Comment:
Not sure if this was intended or not: Before, the class `div` was completely hidden, now the `border-secondary` is always visible, also when there is no description. I would also remove border this since this makes the page scrollable on my 16:9 screen.

Reference:
![image](https://user-images.githubusercontent.com/30267166/161104304-6200d1e2-790b-4262-a3c5-100969afce88.png)

Reply:
Looks perfect now, thanks!